src/components/kytos/misc/StatusMenu.vue

```diff
@@ -0,0 +1,764 @@
+                      </th>
+                      <th>
+                        <input v-model="linkTextFilter[4][1]"></input>
+                      </th>
+                    </tr>
+                  </thead> 
+                  <tbody>
+                    <tr v-for="(item, index) in filtered_linkData" :key="item.id">
+                      <td><k-button class="statusTableButton" :icon="isStoredL(item, index)" v-on:click.native="add_remove_Links(item, index)"></k-button></td>
+                      <td>{{item.name}}</td>
+                      <td :class="statusColor(item)">{{item.status}}</td>
+                      <td>{{item.status_reason | splitStatusReasons}}</td>
+                      <td>{{item.enabled}}</td>
+                      <td>{{item.active}}</td>           
+                    </tr>
+                  </tbody>           
+              </table>
+            </div>
+          </k-accordion-item>
+          <k-accordion-item title="Interfaces">
+            <k-accordion-item title="Filters">
+            <k-button-group>
+              <k-button :class="selectedInterfaceData('ORIGINAL')" icon="database" title="Show-All" tooltip="Show-All" v-on:click.native="changeInterfaceData('ORIGINAL')"></k-button>
+              <k-button :class="selectedInterfaceData('PRE_SELECTED')" icon="clipboard-check" title="Show-Selected" tooltip="Show-Selected" v-on:click.native="changeInterfaceData('PRE_SELECTED')"></k-button>
+              </k-button-group>
+            </k-accordion-item>
+            <div class="data_status_table">
+              <table>
+                <thead>
+                    <tr>
+                      <th rowspan="2"></th>
+                      <th>Switch/Node</th>
+                      <th>Port</th>
+                      <th>Status</th>
+                      <th>Reason</th>
+                      <th>Enabled</th>
+                      <th>Active</th>
+                    </tr>
+                    <tr>
+                      <th>
+                        <input v-model="interfaceTextFilter[0][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="interfaceTextFilter[1][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="interfaceTextFilter[2][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="interfaceTextFilter[3][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="interfaceTextFilter[4][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="interfaceTextFilter[5][1]"></input>
+                      </th>
+                    </tr>
+                  </thead> 
+                  <tbody>
+                    <tr v-for="(item, index) in filtered_interfaceData" :key="item.id">
```

Comment:
Please refer to my comment below regarding opening the InfoPanel to view/edit Interface details.

```suggestion
                    <tr v-for="(item, index) in filtered_interfaceData" :key="item.id" @click="show_infoPanelInterface(item.id)">
```